crates/network/src/any/mod.rs

```diff
@@ -6,14 +6,15 @@ pub use either::{AnyTxEnvelope, AnyTypedTransaction};
 mod unknowns;
 pub use unknowns::{AnyTxType, UnknownTxEnvelope, UnknownTypedTransaction};
 
-pub use alloy_consensus::{AnyHeader, AnyReceiptEnvelope};
+pub use alloy_consensus_any::{AnyHeader, AnyReceiptEnvelope};
 
 use crate::Network;
-use alloy_rpc_types_eth::{AnyTransactionReceipt, Block, Transaction, TransactionRequest};
+use alloy_rpc_types_any::AnyTransactionReceipt;
+use alloy_rpc_types_eth::{Block, Transaction, TransactionRequest};
 use alloy_serde::WithOtherFields;
 
 /// A catch-all header type for handling headers on multiple networks.
-pub type AnyRpcHeader = alloy_rpc_types_eth::Header<alloy_consensus::AnyHeader>;
+pub type AnyRpcHeader = alloy_rpc_types_eth::Header<alloy_consensus_any::AnyHeader>;
```

Comment:
let's move this alias to rpc-types-any

Reply:
Fixed at https://github.com/alloy-rs/alloy/pull/1609/commits/da79415831a3af42cbaca63158aaa8aa484dbb07